stimuli_ci_generation/src/server/main.py

```diff
@@ -45,10 +55,16 @@ def index():
             print(f"error: {msg}")
             return f"Bad Request: {msg}", 400
 
-        print(data["name"])
-        print(data["bucket"])
+        file_name = data["name"]  # file_name should be participant_id.jpg
+        print("file_name:", file_name)
+        participant_id = file_name.split(".")[0]
+        downloaded_path = download_file(
+            RAW_IMG_BUCKET, file_name, f"{mkdir(participant_id)}/{file_name}"
+        )
+        print("downloaded_to:", downloaded_path)
+        # generate_stimuli(downloaded_path, participant_id)
```

Comment:
I'm getting `Fatal error: cannot open file 'generate_stimuli.R': No such file or directory `. Anyone has any idea here?

Reply:
I got this fixed. But now I'm getting Rscript help script, looks like the R call is not correct.

Reply:
Can you post the exact error?

Reply:
it's literally just the Rscript help script, most likely because Rscript was called but was not provided arguments. I'm not entirely sure what happened there

Reply:
Oh okay :+1: